delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphViews.scala

```diff
@@ -379,6 +382,37 @@ object BlazegraphViews {
     ReferenceExchange[BlazegraphView](fetch(_, _), _.source)
   }
 
+  def consistentWrite(
+      client: BlazegraphClient,
+      cache: BlazegraphViewsCache,
+      indexingConfig: ExternalIndexingConfig
+  )(implicit cr: RemoteContextResolution, baseUri: BaseUri): ConsistentWrite = {
+    new ConsistentWrite {
+      override def apply[R](res: ConsistentWrite.ConsistentWriteValue[_, _])(implicit rejectionMapper: Mapper[ConsistentWriteFailed, R]): IO[R, Unit] = {
+        (for {
+          projectViews <- cache.values.map { vs =>
+            vs.filter(v => v.value.project == res.project && v.value.tpe == IndexingBlazegraphViewType)
```

Comment:
The view may also be deprecated.
We also should change the cache to be a `CompositeKeyValueStore` as we do with resolvers so as to be able to go through only the project views